### PR TITLE
refactor(SettingsFaq): code

### LINKED ISSUE #391: settings-faq

### PR DESCRIPTION
**Pull request by samandarkuchkarov**
Fixes #391 

## Proposed Changes

  -Change the imports to absolute:

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

**Comment by samandarkuchkarov** [comment]
CustomHeader is changed in settingsAdressBook

**Comment by samandarkuchkarov** [comment]
CustomHeader is changed in settingsAccountEdit task

File: src/i18n.ts [hunk 1]
```diff
@@ -183,6 +183,10 @@ export enum I18N {
   backupFinishSuccess,
   backupFinishFinish,
   proposalTitle,
+  settingsContactEditDelateContact,
+  settingsContactEditSure,
+  settingsContactEditCancel,
+  settingsContactEditDelete,
```
[the remark this answers is not in the document]
Comment (samandarkuchkarov) [reply src/i18n.ts:186]: done

File: src/i18n.ts [hunk 2]
```diff
@@ -196,6 +196,8 @@ export enum I18N {
   backupFinishSuccess,
   backupFinishFinish,
   proposalTitle,
+  settingsContactEditDelete,
+  settingsContactEditSure,
```
Comment (gHashTag) [inline src/i18n.ts:200]: I meant that the enumeration should go in alphabetical order
image
And in `en` object
image

**Review by gHashTag — CHANGES_REQUESTED** [review]
Fix i18n orders

File: src/screens/settings-contact-edit.tsx [hunk 3]
```diff
@@ -32,21 +32,21 @@ export const SettingsContactEditScreen = () => {
...
-      [
-        {text: 'Cancel', style: 'cancel'},
-        {
-          text: 'Delete',
-          style: 'destructive',
-          onPress: () => {
-            contacts.removeContact(address);
-            goBack();
-          },
+    const DelateContact = getText(I18N.settingsContactEditDeleteContact);
```
Comment (vivalaakam) [inline src/screens/settings-contact-edit.tsx:35]: variables in camelCase pls

Comment (samandarkuchkarov) [reply src/screens/settings-contact-edit.tsx]: done

**Pull request merged**